pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancer.java

```diff
@@ -572,28 +575,28 @@ private RebalanceResult doRebalance(TableConfig tableConfig, RebalanceConfig reb
     }
   }
 
-  private long calculateTableSizePerReplicaInBytes(String tableNameWithType) {
-    long tableSizePerReplicaInBytes = -1;
+  private TableSizeReader.TableSubTypeSizeDetails fetchTableSizeDetails(String tableNameWithType) {
     if (_tableSizeReader == null) {
       LOGGER.warn("tableSizeReader is null, cannot calculate table size for table {}!", tableNameWithType);
-      return tableSizePerReplicaInBytes;
+      return null;
     }
     LOGGER.info("Fetching the table size for rebalance summary for table: {}", tableNameWithType);
```

Comment:
nit: reword this to remove "for rebalance summary" since it will be used for both pre-checks and summary